tcf_website/views/search.py

```diff
@@ -103,6 +107,8 @@ def rank_course(query):
 
 def format_response(response):
     """Formats an Elastic search endpoint response."""
+    if "error" in response:
+        return response
```

Comment:
This currently returns a dictionary, list, or string which seems awkward. Maybe change to always returning a list (empty if error) or returning two values, a list and a boolean/message/etc to signify the specific error raised? This would also make set_arguments simpler as we could do a check of `if courses:` instead of an isInstance check.

Reply:
Great suggestion! I went ahead and changed the `format_response` method to always return an error boolean and results list. Now the return type/format is standardized, which should fix all the edge cases. Go ahead and review the changes below and give the PR another review (or approval if you think its ready). Thank you for taking the initiative on this!